File: Java/lab17_Mediator/task_3_1/src/PlanesOnGround.java
import java.util.ArrayList;

public class PlanesOnGround {

  ArrayList<Plane> planes = new ArrayList<>();

  public void addPlane(Plane plane) {
    planes.add(plane);
  }

  public void removePlane(Plane plane) {
    planes.remove(plane);
  }

}

File: Cpp/lab17_Mediator/task_3_1/src/planes_on_ground.cpp
#include <algorithm>

#include "planes_on_ground.hpp"


PlanesOnGround::PlanesOnGround() {}

void PlanesOnGround::addPlane(Plane* plane)
{
  this->planes.push_back(plane);
}

void PlanesOnGround::removePlane(Plane* plane)
{
  std::remove(this->planes.begin(), this->planes.end(), plane);
}
